### PR TITLE
update migrate

### PR DESCRIPTION
**Pull request by lethu96**
em thiết kế database như thế này. anh review cho em với ạ

### CODE REVIEW

File: app/Member_project.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Member_project extends Model
```
Comment (TaHieu-scuti) [inline app/Member_project.php]: Pls replace Member_project to "MemberProject"

File: app/Member.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Member extends Model
+{
+    //
```
Comment (ngon12) [inline app/Member.php]: Please remove all comment out (if you don't use this after) before you push!

File: app/Member_project.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+{
+    //
```
Comment (ngon12) [inline app/Member_project.php]: Please remove all comment out (if you don't use this after) before you push!

File: app/Member_project.php (hunk 1 above)
Comment (ngon12) [inline app/Member_project.php]: Please change name of Model class like that: `MemberProject` and please change name of Model file like that: `MemberProject.php`.

Comment (lethu96) [reply app/Member_project.php]: yep sir

File: app/Position.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Position extends Model
+{
+    //
```
Comment (ngon12) [inline app/Position.php]: Please remove all comment out (if you don't use this after) before you push!

File: app/Project.php [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Project extends Model
+{
+    //
```
Comment (ngon12) [inline app/Project.php]: Please remove all comment out (if you don't use this after) before you push!

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 6]
```diff
@@ -16,8 +16,10 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
-            $table->string('email')->unique();
+            $table->string('email') ->unique();
```
Comment (TaHieu-scuti) [inline database/migrations/2014_10_12_000000_create_users_table.php]: mũi tên viết liền không cách

File: database/migrations/2018_01_17_024316_create_positions_table.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('positions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('description') ->nullable();
```
Comment (TaHieu-scuti) [inline database/migrations/2018_01_17_024316_create_positions_table.php]: mũi tền viết liền không có cách

**Review by ngon12 — COMMENTED** [review]
I have some request to you, can you check it? Thanks

File: database/migrations/2018_01_17_024345_create_members_table.php [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('members', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('information') -> nullable();
```
Comment (TaHieu-scuti) [inline database/migrations/2018_01_17_024345_create_members_table.php]: giống lỗi ở trên, fix nó giúp a

File: database/migrations/2018_01_17_024500_create_member_projects_table.php [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('member_projects', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('meber_id');
+            $table->integer('project_id');
+            $table->string('role') -> nullable();
```
Comment (TaHieu-scuti) [inline database/migrations/2018_01_17_024500_create_member_projects_table.php]: cả đây nữa

File: database/migrations/2018_01_17_024549_create_projects_table.php [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('projects', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('information') -> nullable();
```
Comment (TaHieu-scuti) [inline database/migrations/2018_01_17_024549_create_projects_table.php]: giống lỗi trên

File: app/Member.php (hunk 2 above)
Comment (lethu96) [reply app/Member.php]: ok

File: app/Member_project.php (hunk 1 above)
Comment (lethu96) [reply app/Member_project.php]: ok  man

File: app/Member_project.php (hunk 3 above)
Comment (lethu96) [reply app/Member_project.php]: oke man

File: app/Position.php (hunk 4 above)
Comment (lethu96) [reply app/Position.php]: oke man

File: app/Project.php (hunk 5 above)
Comment (lethu96) [reply app/Project.php]: oke man

**Review by TaHieu-scuti — APPROVED** [review]

**Review by ngon12 — APPROVED** [review]
OK for me

**Pull request merged**